### PR TITLE
Ignore up-to-date collected trackers

### PR DESCRIPTION
**Pull request by costaconrado**
This PR make Trackers not run validations for the parent Flaw.

This PR also adds a manual call to duplication validation in Tracker::save method to avoid having two equal entries in BTS in case multiple workers use the same entity at the same time.

This PR also changes how collectors and converters work for Trackers, only calling the `save` method in case a Tracker doesn't already have the latest BTS timestamp.

This PR also do changes in serializer so Trackers are created with default fake date (1/1/1970) that will be replaced the first time the collectors or the sync manager get the BTS timestamp.

Closes OSIDB-3244.
Closes OSIDB-3234.

### CODE REVIEW

File: osidb/serializer.py [hunk 1]
```diff
@@ -687,8 +688,12 @@ def create(self, validated_data):
 
         # create new tracker instance in the local DB first
         # so we can create the links before the backend sync
-        # alerts will be created after links
-        tracker.save(no_alerts=True)
+        # alerts will be created after links and timestamp
+        # should be generated by BTS and will be fixed during sync
```
Comment (jsvob) [inline osidb/serializer.py:692]: Nitpick: Is there an extra word in the comment?

**Review by jsvob — APPROVED** [review]
Very nice! LGTM! Thank you!

Didn't test it and someone else should review as well.

File: osidb/models.py [hunk 2]
```diff
@@ -2424,6 +2424,10 @@ def save(self, *args, bz_api_key=None, jira_token=None, **kwargs):
             and bz_api_key is not None
             and self.type == self.TrackerType.BUGZILLA
         ):
+            # avoid creating creating tracker in duplicity
```
Comment (osoukup) [inline osidb/models.py:2427]: nitpick: double `creating`

**Review by osoukup — APPROVED** [review]
LGTM. Good job!

File: osidb/serializer.py (hunk 1 above)
Comment (costaconrado) [reply osidb/serializer.py:692]: Message improved, thanks.

File: osidb/models.py (hunk 2 above)
Comment (costaconrado) [reply osidb/models.py:2427]: Duplicated word removed, thanks.

**Pull request merged**